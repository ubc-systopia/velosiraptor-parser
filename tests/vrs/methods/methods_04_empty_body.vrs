segment MyUnit {
    // method with pre-condition
    fn foo() -> bool
    {

    }
}