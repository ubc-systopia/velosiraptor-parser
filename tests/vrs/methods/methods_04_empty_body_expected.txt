error: expected one of `~`, `!`, `integer literal`, `boolean literal`, `identifier`, `if`, `exists`, `forall`, or `(`, found `}`
     --> tests/vrs/methods/methods_04_empty_body.vrs:6:5
      |
    6 |      }
      |      ^ expected one of 9 possible tokens
